tutorials/on-beyond-magpie0.md

```diff
@@ -0,0 +1,111 @@
+
+Not only does it recognize that "was" is a verb, it returns the base verb "be" and the mood, number, person, and tense of the verb.
+
+### Sentiment Analysis
+Sentiment analysis returns a score from -1.0 (for negative) to 1.0 (for positive) sentiment.  The magnitude is the strength of the sentiment.  If the text consists of more than one sentence, there will be a score for the entire text and each individual sentence.  Encourage students to try to create sentences with high magnitude. 
+
+## Accessing the API with the API Explorer
+
+The interactive feature of the WWW page does not show the JSON value returned by the API.  To see the returned JSON, use the [API Explorer][explorer].  You can select to analyze any one item (entities, sentiment, or syntax) or all three ([annotateText][annotate]).  (The pages describing many of the other APIs also have a "TRY IT!" button that will access the API Explorer directly.)
+
+This provides a way to introduce students to HTTP calls.  The details of the call parameters can be found [on-line][annotateapi].  In the API explorer, the context menu will show the properties that can be used.  As a minimum, in the `Request` body:
+
+* Add a `document` property.  Within the `document` property. 
+  * Add a `type` property with the value  `PLAIN_TEXT`.
+  * Add a `content` property with the text to be examined.
+* Add an `encodingType` property with the value `UTF8`.
+* Add a `features` property.  Within the `features` property.
+  * Add at least one of the properties `extractDocumentSentiment`, `extractEntities` and `extractSyntax` and check the box to set the property to true.
+
+Use the `Execute` button to execute the API call.
+
+The `Response` data will appear below the call in JSON format.
+
+## Summary
+
+* If you want to use the Cloud Natural Language API in a program, you will need to:
+
+  * Create a GCP project.
+  * Enable the API
+  * Arrange for authentication (in this case using an API key).
+
+* From the web interface, you can use the Cloud Natural Language API to analyze sentiment, extract entities, and parse the syntax of text.
+* You can also call the Cloud Natural Language API from the API Explorer.
+
+## Next Steps 
+To use the different features of the Cloud Natural Language, see the following Community articles:
+* [On Beyond Magpie: Part 1, Sentiment Analysis][magpie1]
+* [On Beyond Magpie: Part 2, Entity Analysis][magpie2]
+
+
+
+
+
+[alice]:https://www.gutenberg.org/files/11/11-h/11-h.htm
+[annotate]:https://apis-explorer.appspot.com/apis-explorer/#search/natural/language/v1/language.documents.annotateText
+[annotateapi]:https://cloud.google.com/natural-language/docs/reference/rest/v1beta1/documents/annotateText
+[auth]:https://cloud.google.com/natural-language/docs/common/auth
+[console]:https://console.cloud.google.com/
```

Comment:
I usually create projects from the dashboard, so as is is fine for me, but you've got more experience.